src/irr/asset/CPLYMeshWriter.cpp

```diff
@@ -448,7 +446,7 @@ asset::ICPUMeshBuffer* CPLYMeshWriter::createCopyMBuffNormalizedReplacedWithTrue
             mbCopy->getPipeline()->getVertexInputParams().attributes[vaid].format = asset::isNormalizedFormat(t) ? impl::getCorrespondingIntegerFormat(t) : t;
     }
 
-    return mbCopy;
+    return mbCopy.get();
```

Comment:
return the smart pointer instead, don;t do weird grabs and stuff

Reply:
okay

Reply:
return type of function still not changed

Reply:
oh sorry, to fix